src/content/self-hosted/administration/certificates.mdx

```diff
@@ -32,8 +32,13 @@ ingress-nginx:
       default-ssl-certificate: $(POD_NAMESPACE)/your-ssl-certificate-secret
 ```
 
-In order to generate your certificates with [cert-manager](https://cert-manager.io/) you need to use a [DNS01](https://cert-manager.io/docs/configuration/acme/dns01/#delegated-domains-for-dns01) auth method in your [Issuer](https://cert-manager.io/docs/concepts/issuer/).
+If you are using [cert-manager](https://cert-manager.io/), you need to use a [DNS01](https://cert-manager.io/docs/configuration/acme/dns01/#delegated-domains-for-dns01) auth method in your [Issuer](https://cert-manager.io/docs/concepts/issuer/).
```

Comment:
That link points to a subsection of the dns01 docs, instead link to the main doc (remove the # part)